.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: release
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  npm-publish:
+    name: Publish to NPM & GitHub Package Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: main
+      # limit releases to version changes - https://github.com/EndBug/version-check
+      - name: Check version changes
+        uses: EndBug/version-check@v1
+        id: version_check
+        with:
+          file-url: https://unpkg.com/@grafana/aws-sdk@latest/package.json
```

Comment:
URL should point to `@grafana/async-query-data` package

Reply:
True lol 🤦🏻‍♀️